cmd/crowdsec-cli/metrics.go

```diff
@@ -329,6 +352,30 @@ func ShowPrometheus(url string) {
 			}
 		}
 
+		decisionsTable := tablewriter.NewWriter(os.Stdout)
+		decisionsTable.SetHeader([]string{"Scenario", "Origin", "Action", "Hits"})
```

Comment:
if it's a counter of active decisions, we might find something more appropriate than `Hits`. `Count` or `Current` or something around that maybe ?